.github/workflows/packages-build.yml

```diff
@@ -7,17 +7,31 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        package_type: ["deb", "rpm"]
-        python-version: [3.8]
-        os: [ubuntu-20.04]
         include:
-          - os: ubuntu-20.04
-            package_type: "tar"
+          - package_type: "deb"
+            package_name: "scalyr-agent-2_*.*.*_all.deb"
```

Comment:
Minor thing - might be better to call this ``package_filename_glob`` to make it obvious it expects / can take a glob pattern.